src/property-filter/__tests__/property-filter-token-editor.test.tsx

```diff
@@ -280,3 +292,308 @@ describe.each([false, true])('token editor, expandToViewport=%s', expandToViewpo
     });
   });
 });
+
+const externalProperty = {
```

Comment:
All tests below were copied from the token-editor-grouped test file, no changes to the tests were made